examples/lists.rs

```diff
@@ -0,0 +1,49 @@
+#[macro_use]
+extern crate redis_module;
+
+use redis_module::raw::KeyType;
+use redis_module::{Context, RedisError, RedisResult, RedisValue};
+
+// LPOPRPUSH source destination
+// Pops and returns the first element (head) of the list stored at 'source'
+// and pushes the element to the last position (tail) of the list stored at
+// 'destination'.
+fn lpoprpush(ctx: &Context, args: Vec<String>) -> RedisResult {
+    if args.len() != 3 {
+        return Err(RedisError::WrongArity);
+    }
+
+    let src_key = ctx.open_key_writable(&args[1]);
+    let dst_key = ctx.open_key_writable(&args[2]);
```

Comment:
It would be nice to reuse the existing argument parsing and error handling code, which is more idiomatic in Rust (since it unifies the parsing and error handling instead of separating them):
```suggestion
    let mut args = args.into_iter().skip(1);
    
    let src = args.next_string()?;
    let dst = args.next_string()?;

    let src_key = ctx.open_key_writable(&src);
    let dst_key = ctx.open_key_writable(&dst);
```